src/components/RewardCards/withdraw.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from 'react'
+import { useActiveWeb3React } from '../../hooks'
+import { ButtonPrimary } from '../Button'
+import EstimatedRewards from './modal'
+import Percentage from './percentage'
+import styled from 'styled-components'
+import { withdrawLP } from '../../utils/rewards'
+import { Reward, User } from '../../utils/farm/constants'
+
+const Container = styled('div')`
+  text-align: left;
+  display: flex;
+  flex-wrap: wrap;
+`
+
+const Wrapper = styled('div')`
+  display: flex;
+  flex: wrap;
+  padding-bottom: 14px;
+  margin: auto;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  width: 100%;
+  overflow: hidden;
+  text-align: left;
+  justify-content: flex-end;
+`
+
+const InputWrapper = styled('div')`
+  display: flex;
+  flex: wrap;
+  margin: auto;
+  border-radius: 12px;
+  padding: 12px;
+  border: 2px solid white;
+  width: 100%;
+  overflow: hidden;
+  text-align: left;
+  justify-content: flex-end;
+  > span {
+    margin: auto;
+  }
+`
+const Input = styled('input')`
+  display: flex;
+  flex: 1;
+  font-size: 16px;
+  background: none;
+  border: none;
+  padding: 0.5rem;
+  color: white;
+  :focus {
+    outline: none;
+  }
+`
+const Text = styled('div')`
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 10px;
+  text-align: right;
+  color: #b5b9d3;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 18px;
+`
+
+const Balance = styled('div')`
+  display: flex;
+  text-align: center;
+  font-size: 14px;
+  font-weight: 500;
+  line-height: 18px;
+`
+
+interface RewardProps {
+  data: User
```

Comment:
i think we need better naming here, data should be user and contract can be reward